blocks/lead-art-block/themes/news.json

```diff
@@ -113,35 +51,40 @@
 			}
 		}
 	},
-	"lead-art-carousel-track-icon": {
+	"lead-art-track-button": {
```

Comment:
In the `_index.scss` this is called `lead-art-carousel-track-button`, so the styles aren't loading. Either name is fine, but they need to match in both files.

Reply:
Good catch! It is solved.
Thank you!